pom.xml

```diff
@@ -173,6 +173,11 @@ THE SOFTWARE.
       <artifactId>wiremock</artifactId>
       <version>2.1.12</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
```

Comment:
I need minimally guava-18.0 and the version we're pulling in through Jenkins core is guava-11.0.1.  Not 100% sure how to work around this at the moment; I might have to rethink this.